packages/widget/src/hooks/useAutoSetAddress.ts

```diff
@@ -181,10 +173,7 @@ export const useAutoSetAddress = () => {
               }
               if (!wallet) {
                 if (!openModal) return;
-                NiceModal.show(Modals.SetAddressModal, {
-                  signRequired: isSignRequired,
-                  chainId: chainID,
-                });
```

Comment:
forgot to pass index here :( 
I made a function to handle this so we dont have to worry about remembering to pass index through